cbmc/proofs/coeff_signed_to_unsigned/coeff_signed_to_unsigned_harness.c

```diff
@@ -0,0 +1,30 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
```

Comment:
Why the License header is MIT-0 in this file only?

Reply:
Goot spot - corrected to Apache-2.0